### ISSUE TITLE
Downloads not visible

### ISSUE THREAD

**Issue by gregorko** [SkyTubeTeam/SkyTube#917]
I started a download of videos. After the downloads are finished the videos are not visible in the download tab.
I checked in the directory the videos are stored in. They contain the downloaded files. But there are still not visible in the download tab.
I gave all permissions to the app that I could find.

Xiaomi Poco X3
Not rooted
App from FDorid v2.977

**Comment by gzsombor** [comment]
Yes, there are a couple of problems (bugs and missing features) with the current downloads, at least:
#343 , #389, #529, #739, #886,
